<launch>

  <node pkg ="my_sphero_main"
        type="my_sphero_main.py"
        name="my_sphero_main"
        output="screen">
  </node>
  
  <node pkg="my_sphero_topics"
        type="my_sphero_topics.py"
        name="my_sphero_topics"
        output="screen">
  </node>
  
  <node pkg ="my_sphero_services"
        type="my_sphero_services_server.py"
        name="my_sphero_services_server"
        output="screen">
  </node>
  
  <node pkg ="my_sphero_services"
        type="my_sphero_services_client.py"
        name="my_sphero_services_client"
        output="screen">
  </node>

</launch>